reports/stability-summary/connection-details.txt.sample

```diff
@@ -0,0 +1 @@
+host=something.something dbname=pipeline user=read_only password=AAAABBBB
```

Comment:
You should probably fetch these credentials from the metadata stored in S3.  See the [derived_streams](https://github.com/mozilla-services/data-pipeline/blob/master/reports/derived_streams/run.sh#L39) job for an example on fetching the metadata and credentials.